plugins/org.jboss.tools.forge.ui/META-INF/MANIFEST.MF

```diff
@@ -11,8 +11,8 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.jdt.launching,
  org.eclipse.ui.editors,
  org.eclipse.ui.ide,
- org.eclipse.m2e.core;bundle-version="[1.0,2.0)",
- org.eclipse.m2e.maven.runtime;bundle-version="[1.0,2.0)",
+ org.eclipse.m2e.core;bundle-version="1.0.0",

```

Comment:
Does not seems logical to me the lower bound should be 2.0 or 2.0.0 ?

Reply:
correct. as m2e major changed i should reflect this in MANIFEST.